build.go

```diff
@@ -54,19 +54,60 @@ func Build(procs ProcMgr, logger scribe.Emitter) packit.BuildFunc {
 			return packit.BuildResult{}, errors.New("need exactly one of: $PHP_HTTPD_PATH or $PHP_NGINX_PATH")
 		}
 
+		shouldEnableReload, err := reloader.ShouldEnableLiveReload()
+		if err != nil {
+			return packit.BuildResult{}, err
+		}
+
 		logger.Process("Determining start commands to include in procs.yml:")
+		watchPaths := make([]string, 0)
 		// HTTPD Case
 		if httpdConfPath != "" {
 			serverProc := NewProc("httpd", []string{"-f", httpdConfPath, "-k", "start", "-DFOREGROUND"})
+
+			if exists, err := fs.Exists(filepath.Join(context.WorkingDir, ".httpd.conf.d")); err != nil {
+				return packit.BuildResult{}, err
+			} else if shouldEnableReload && exists {
+				serverProc = NewProc("watchexec", []string{
+					"--watch", "/workspace/.httpd.conf.d",
+					"--on-busy-update", "signal",
+					"--signal", "SIGHUP",
+					"--", "httpd",
+					"-f", httpdConfPath, "-k",
+					"start",
+					"-DFOREGROUND",
+				})
+			} else if shouldEnableReload && !exists {
+				logger.Subprocess("HTTPD will not be reloadable since .httpd.conf.d folder not found")
+			}
+
 			procs.Add("httpd", serverProc)
 			logger.Subprocess("HTTPD: %s %v", serverProc.Command, strings.Join(serverProc.Args, " "))
+			watchPaths = append(watchPaths, "/workspace/.httpd.conf.d")
```

Comment:
Why add this path to the watch path in the case where this directory doesn't exist? Shouldn't this only happen in the `shouldEnableReload && exists` case?